api/v1alpha1/condition_consts.go

```diff
@@ -37,6 +37,14 @@ const (
 	VirtualMachineImageNotFoundReason = "VirtualMachineImageNotFound"
 )
 
+const (
+	// DeployedCondition exposes the deployment status of a VirtualMachine on the infrastructure provider.
+	DeployedCondition ConditionType = "Deployed"
```

Comment:
I'm not sure I understand what this means. The VM hardware deployed successfully? The virtual hardware failed? The guest customization succeeded / failed? Both? I am not sure this makes sense as-is. Let's revisit this a bit more. I don't know that aggregate conditions make sense, and we already have a guest customization condition, so is deploy separate or a superset?

Reply:
Deploy is separate, It's synonymous the Created/Create in VM operator. 

guest customization is triggered in our Update path after Create/Deploy passes.

Reply:
> Deploy is separate, It's synonymous to Created/Create in VM operator.

The issue is that "deploy" is vague enough to be taken multiple ways. Plus, [conditions exist](https://maelvls.dev/kubernetes-conditions/) as a way to be more distinct than allowed by a top-level status field such as `phase`. So let's come up with a better word than `deployed` as it seems to generic and could be mistaken to mean guest customization as well.